syncopy/specest/freqanalysis.py

```diff
@@ -145,15 +145,15 @@ def freqanalysis(data, method='mtmfft', output='pow',
         but may be unbounded (e.g., ``[-np.inf, 60.5]`` is valid). Edges `fmin`
         and `fmax` are included in the selection. If `foilim` is `None` or
         ``foilim = "all"``, all frequencies are selected.
-    pad_to_length : int, None or 'nextpow2'
+    pad : 'maxperlen', float or 'nextpow2'
```

Comment:
Same here - took the liberty and rephrased based on the docstring of `process_padding`

Reply:
thanks, I clearly overlooked this.. 👍